scripts/vendor_prost.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "${SCRIPTS_DIR}/common"
+
+# Updates our local copy of https://github.com/danburkert/prost and applies a small patch to add
+# support for the message_type field option.
+
+# Master branch as of 2020-06-05
+PROST_ZIP_URL="https://github.com/danburkert/prost/archive/80256277997975948d257faf3f35c2890bf12787.zip"
```

Comment:
Perhaps make the commit id a separate variable and reuse here and below, so it's the only thing we need to change when updating.

Also make it consistent with what we do in Dockerfile:

https://github.com/project-oak/oak/blob/56e2996ac20819d8807ef538aebe116922fec752/Dockerfile#L143-L152

Reply:
Done.